src/ListMenu/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { View } from '../../src'
+
+const ListMenu = ({
+   activeBackgroundColor,
+   activeTextColor,
+   children,
+   buttonBorderRadius,
+   buttonPaddingHorizontal,
+   buttonSize,
+   dividerLineStyle,
+   showActiveArrow,
+   textColor
+}) => {
+   // Clone ListMenuButton children and pass props
+   const listMenuButtons = React.Children.map(children, (child, i) => {
+      // This is to handle the cases where if buttons are mapped, there may be null values
+      if (child) {
+         return (
+            <>
+               {React.cloneElement(child, {
+                  activeBackgroundColor,
+                  activeTextColor,
+                  showActiveArrow,
+                  textColor,
+                  borderRadius: buttonBorderRadius,
+                  paddingHorizontal: buttonPaddingHorizontal,
+                  size: buttonSize
+               })}
+               {i < (children.length - 1) && (
+                  <View
```

Comment:
Any reason not to just use a plain `div`? Is it because the divider is a responsive string?